rest/client/methods.go

```
@@ -1111,7 +1112,15 @@ func (c *communicatorImpl) GetCommitQueue(ctx context.Context, projectID string)
 	return &cq, nil
 }
 
-func (c *communicatorImpl) DeleteCommitQueueItem(ctx context.Context, projectID, item string) error {
+func (c *communicatorImpl) DeleteCommitQueueItem(ctx context.Context, item string) error {
+	requestedPatch, err := patch.FindOneId(item)
+	if err != nil {
+		return errors.Wrap(err, "error finding item")
+	}
+	if requestedPatch == nil {
+		return errors.Wrap(err, "item not found")
```

Since error is nil here you can just return an error with the message only.